### PR TITLE
Update command line dokku recommendations in favor of DE GraphQL API

### PR DESCRIPTION
**Pull request by chriddyp**
Fixes https://github.com/plotly/streambed/issues/15503
Post-merge checklist:

The master branch is auto-deployed to `dash.plotly.com`.
Once you have merged your PR, wait 5-10 minutes and check dash.plotly.com
to verify that your changes have been made.

- [x] I understand

If this PR documents a new feature of Dash:

- [ ] Comment on the original Dash issue with a link to the new docs.
- [ ] Reply to any community thread(s) asking for this feature.

If this PR includes a new dataset available at a remote URL:
- [ ] I have added this dataset to the `datasets/` folder
- [ ] I have added a mapping between the remote URL and the filename in the
`datasets/` folder into the `find_and_replace` dict in `dash_docs/tools.py`

If this PR adds an image or animated GIF:
- [ ] This image was saved and referenced locally rather than via an external link

If I introduced a new relative link inside `dcc.Markdown`:
- [ ] I considered whether I could replace the `dcc.Markdown` call with `rc.Markdown`, which will replace relative links with `tools.relpath` internally. Otherwise, I used e.g. `<dccLink href=tools.relpath('/layout') children="the first chapter"/>` instead of `[the first chapter](/layout)` (importing `tools` from `dash_docs`), and set `dangerously_allow_html=true` in the `dcc.Markdown` call.

If I changed the `chapter_index` by removing or relocating a page:
- [ ] I added a redirect in `dash_docs/server.py` from the old URL to the new URL

### CODE REVIEW

File: dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py [hunk 1]
```diff
@@ -1468,11 +1468,24 @@ def display_instructions(platform):
...
+    html.H1('Managing Dash Apps via Dokku instead of Official API'),
 
     Blockquote(),
 
     rc.Markdown('''
+    Certain management actions are not yet exposed in the official 
+    <dccLink href="/dash-enterprise/api" children="Dash Enterprise GraphQL API"/>.
+    
+    Until these commands are exposed in the API,
+    you can run a SSH commands against a core part of the Dash Enterprise
```
Comment (wbrgss) [inline dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py:1480]:
```suggestion
    you can run SSH commands against a core part of the Dash Enterprise
```

File: dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py [hunk 2]
```diff
@@ -1988,44 +1952,33 @@ def display_instructions(platform):
...
-
-        `ssh dokku@your-dash-enterprise -p PORT redis:link redis-db my-dash-app`
-
-        &nbsp;
-    ''')
+    dcc.Markdown([
+        '''
+        ### Unsupported and Deprecated Commands
+        
+        These commands are no longer supported or recommended.
```
Comment (wbrgss) [inline dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py:1959]:
```suggestion
        These `dokku` commands are no longer supported or recommended. Each of their recommended alternatives, respectively, are provided below.
```

Non-blocking, I'm just thinking that a user might see "unsupported and deprecated" and skip the entire section, or get confused and think all of the below commands are deprecated, at least until they read enough to get the context.

File: dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py [hunk 3]
```diff
@@ -2287,9 +2240,11 @@ def update_graph(value):
     Redis is a powerful in-memory database that is well-suited for many Dash
     applications. In particular, you can use Redis to:
 
-    - Save application data
+    - Save application data that persists in memory across processes, containers, and deploys.
     - Enable queued and background processes with Celery.
     [Redis and Celery Demo App](https://github.com/plotly/dash-redis-demo)
+    - Use the Dash Enterprise Snapshot Engine, see the documentation on your
```
Comment (wbrgss) [inline dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py:2246]:
```suggestion
    - Use the Dash Enterprise Snapshot Engine; see the documentation on your
```

File: dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py [hunk 4]
```diff
@@ -2414,34 +2347,11 @@ def update_graph(value):
...
-
-    &nbsp;
-
-    In the commands above, replace:
-    * `YOUR_DASH_SERVER` with the name of your Dash server
-    (same as when you run `git remote add`)
-    * `SERVICE-NAME` with the name you want for your Redis service
-    * `APP-NAME` with the name of your app (as specified in the
-    Dash App Manager).
+    database via the [Dash Enterprise GraphQL API](/dash-enterprise/api)
```
Comment (wbrgss) [inline dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py:2354]:
```suggestion
    database via the [Dash Enterprise GraphQL API](/dash-enterprise/api).
```

File: dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py (hunk 1 above)
Comment (wbrgss) [reply dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py:1480]: This is the only actual blocking suggestion from me

**Review by wbrgss — APPROVED** [review]
Some copy suggestions, otherwise LGTM @chriddyp. I'm not very familiar with this DE feature so I can't comment on the viability of the deprecations/workarounds if you want to wait for a tagged review instead. But they look like they're in the right place.

:woman_dancing: after considering my comments below

**Comment by wbrgss** [comment]
Also just a note, seems like the Percy baseline for `master` is out of date. 

E.g. the gif at the bottom of the page in http://dash.plotly.com/authentication is already published, and the

> This user guide is for the Python implementation of Dash Open Source. 

message on the homepage doesn't show up on the Percy `master` baseline, but shows as published on http://dash.plotly.com

**Comment by chriddyp** [comment]
Excellent review, thank you @wbrgss !

**Review by alexcjohnson — APPROVED** [review]

**Pull request merged**